TESTING.md

```diff
@@ -0,0 +1,7 @@
+# Vérifier l’application
+
+L’application bénéficie de tests frontend, backend et d’intégration (end-to-end). L’utilisation de chaque suite de test eest documenté dans le dossier associé :
```

Comment:
```suggestion
L’application bénéficie de tests frontend, backend et d’intégration (end-to-end). L’utilisation de chaque suite de test est documenté dans le dossier associé :
```

Reply:
je pense que ca peut etre utile d'avoir des infos récapitulative ici (comment lancer les tests de app / backend / e2e. Si le fichier ne fait que lister 3 autres fichiers, je suis pas sur de son utilité

Reply:
Bonne remarque. J’ai déplacé les sections de tests frontend et backend directement dans TESTING.md. Comme la documentation des tests end-to-end est plus complète, j’ai laissé un lien dans `TESTING.md` et le document `e2e/README.md`.

Reply:
top merci